carma/rviz/carma_default_ros2.rviz

```diff
@@ -0,0 +1,525 @@
+Panels:
+  - Class: rviz_common/Displays
+    Help Height: 138
+    Name: Displays
+    Property Tree Widget:
+      Expanded:
+        - /Global Options1
+        - /Status1
+        - /LaneLet2 Map1/Topic1
+        - /LaneLet2 Map1/Namespaces1
+        - /Trajectory Visualizer1/Topic1
+        - /Points Map1
+        - /Points Map1/Topic1
+        - /TCM Visualizer1/Topic1
+        - /Route Marker1
+        - /Route Marker1/Topic1
+        - /Host MobilityPath Marker1/Topic1
+        - /Host MobilityPath Marker1/Namespaces1
+        - /GNSS Pose1/Topic1
+        - /Pose1
+        - /Vehicle Model1
+        - /Vehicle Model1/Description Topic1
+        - /Vehicle Model1/Links1
+      Splitter Ratio: 0.5
+    Tree Height: 1216
+  - Class: rviz_common/Selection
+    Name: Selection
+  - Class: rviz_common/Tool Properties
+    Expanded:
+      - /Interact1
+      - /Measure1
+      - /2D Pose Estimate1
+      - /2D Pose Estimate1/Topic1
+      - /2D Goal Pose1
+      - /2D Goal Pose1/Topic1
+      - /Publish Point1
+    Name: Tool Properties
+    Splitter Ratio: 0.5886790156364441
+  - Class: rviz_common/Views
+    Expanded:
+      - /Current View1
+    Name: Views
+    Splitter Ratio: 0.5
+  - Class: rviz_common/Transformation
+    Name: Transformation
+Visualization Manager:
+  Class: ""
+  Displays:
+    - Alpha: 0.5
+      Cell Size: 1
+      Class: rviz_default_plugins/Grid
+      Color: 160; 160; 164
+      Enabled: false
+      Line Style:
+        Line Width: 0.029999999329447746
+        Value: Lines
+      Name: Grid
+      Normal Cell Count: 0
+      Offset:
+        X: 0
+        Y: 0
+        Z: 0
+      Plane: XY
+      Plane Cell Count: 10
+      Reference Frame: <Fixed Frame>
+      Value: false
+    - Class: rviz_default_plugins/MarkerArray
+      Enabled: true
+      Name: LaneLet2 Map
```

Comment:
In this case, the Queue Size of 100 means that RViz will keep up to 100 MarkerArray messages in its queue. Since we are only publishing lanelet map once, it shouldn't affect it.